collectors/template/test/test.js.template

```diff
@@ -0,0 +1,121 @@
+const assert = require('assert');
```

Comment:
Not sure this test template makes sense.
I think unit test for a collector mostly contains for `_getNextCollectionState()` and `pawsGetLogs()`